app/users/resources.py

```diff
@@ -0,0 +1,48 @@
+from flask import request
+from flask_restful import Resource
+import json
+
+from app.users.models import User
```

Comment:
Nit. `User` is actually imported in our app's root `__init__.py` file, so you could simplify this and the line below to 
`from app import db, User`